### DISCUSSION TITLE
Add to the report the settings of the pg_profile with which it was launched.

### DISCUSSION THREAD

**Discussion by Sshumeev** [zubkov-andrei/pg_profile#107] [Ideas]
Add to the report the settings of the pg_profile with which it was launched. Is it possible to view the settings of the pg_profile by a query? For example, in pscl?

**Comment by zubkov-andrei** [comment]
Hi! pg_profile settings, set in postgresql.conf file will be included in the report. If they are not listed, then the defaults was used.

**Comment by Sshumeev** [comment]
Hi! Is it possible to always include pg_profile settings specified in the postgresql.conf file in the report? It is not convenient to look for default values somewhere + they can be different for different versions of pg_profile + many users do not know about the existence of these settings.

**Reply by zubkov-andrei** [reply]
It seems it will require a some changes in the settings collection functions. This issue may be related to other settings undefined by postgres and its extensions.
